Addind information #3198 (#3234)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md

- [The maximum number of concurrent shells for this tenant in Skype for Business Online has been exceeded](#the-maximum-number-of-concurrent-shells-for-this-tenant-in-skype-for-business-online-has-been-exceeded)
    

> [!IMPORTANT]

> > Powershell sessions by default stand for sixty minutes, for reconnecting, you have to close and launch a new Powershell session to reconnect. Recently has been launched a new version of [Skype for Business Online, Windows PowerShell Module (2046.123 - Published 10/2/2019)](https://www.microsoft.com/en-us/download/details.aspx?id=39366) which brings a new Cmdlet called **Enable-CsOnlineSessionForReconnection** which mitigate the 60 minutes timeout issue.

Reviewer: ```suggestion
> By default, PowerShell sessions time out after sixty minutes. To reconnect, you have to close the session and launch a new PowerShell session. A new version of [Skype for Business Online, Windows PowerShell Module (2046.123 - Published 10/2/2019)](https://www.microsoft.com/download/details.aspx?id=39366), has been launched recently which includes a new cmdlet called **Enable-CsOnlineSessionForReconnection** that mitigates the 60 minutes time-out issue.
```

> > The PS session reconnects and authenticates, allowing it to be re-used without having to launch a new instance to reconnect.

Reviewer: ```suggestion
> The PowerShell session reconnects and authenticates, allowing it to be re-used without having to launch a new instance to reconnect.
```
